spec/features/shelterreviews/shelter_review_edit_cont_spec.rb

```diff
@@ -0,0 +1,99 @@
+require 'rails_helper'
```

Comment:
It would be nice if there was a better way to write this test a little cleaner, but it works.